### PR TITLE
Task/apps 2983 webpack 5

### PR DESCRIPTION
**Pull request by solomontang**
- update dependencies to use webpack 5
- bundle size reduced to 272KiB from 4.13MiB
  - before
image

  - after 
image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,94 +30,129 @@
...
-    "@storybook/react": "~5.3.19",
+    "@babel/cli": "~7.14.0",
+    "@babel/core": "~7.14.0",
+    "@babel/plugin-proposal-class-properties": "~7.13.0",
+    "@babel/plugin-proposal-object-rest-spread": "~7.14.4",
+    "@babel/plugin-transform-runtime": "~7.14.3",
+    "@babel/preset-env": "~7.14.4",
+    "@babel/preset-flow": "~7.13.13",
+    "@babel/preset-react": "~7.13.13",
+    "@babel/register": "~7.13.16",
```
Comment (UnsungHero97) [inline package.json:47]: You can remove this dependency.

File: package.json [hunk 2]
```diff
@@ -30,94 +30,129 @@
...
+    "connected-react-router": "~6.9.0",
+    "core-js": "~3.12.0",
     "enzyme": "~3.11.0",
-    "enzyme-adapter-react-16": "~1.15.0",
+    "enzyme-adapter-react-16": "~1.15.6",
     "eslint": "~7.27.0",
-    "eslint-config-airbnb": "~18.2.0",
+    "eslint-config-airbnb": "~18.2.1",
     "eslint-import-resolver-webpack": "~0.13.0",
     "eslint-loader": "~4.0.0",
```
Comment (UnsungHero97) [inline package.json:75]: You can remove this dependency, it's not used anywhere.

File: package.json [hunk 3]
```diff
@@ -30,94 +30,129 @@
...
+    "@redux-saga/core": "^1.1.0",
+    "connected-react-router": "^6.9.0",
+    "history": "^4.10.0",
+    "immutable": "4.0.0-rc.10",
+    "lattice": "^0.92.0",
+    "lattice-auth": "^0.23.0",
+    "lattice-fabricate": "^0.17.0",
+    "lattice-sagas": "^0.66.0",
+    "lattice-ui-kit": "~0.43.0",
+    "lattice-utils": "~0.11.0",
```
Comment (UnsungHero97) [inline package.json:140]: Why mix `~` and `^`?

File: package.json [hunk 4]
```diff
@@ -30,94 +30,129 @@
...
+    "lodash": "^4.17.21",
+    "luxon": "^1.26.0",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
+    "react-redux": "^7.2.0",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
+    "redux": "^4.0.0",
+    "redux-immutable": "4.0.0",
+    "redux-reqseq": "~0.6.0",
```
Comment (UnsungHero97) [inline package.json:150]: Should this be `~` or `^`?

File: src/containers/restitutionreferral/RestitutionReferral.js [hunk 5]
```diff
@@ -203,7 +203,7 @@ const RestitutionReferral = () => {
     ]);
 
     const entityData = processEntityData(formDataForSubmit, entitySetIds, propertyTypeIds);
-    const associations = [
+    const associations :any[][] = [
```
Comment (UnsungHero97) [inline src/containers/restitutionreferral/RestitutionReferral.js:206]: Are all of these really necessary?

**Review by UnsungHero97 — CHANGES_REQUESTED** [review]

File: src/containers/restitutionreferral/RestitutionReferral.js (hunk 5 above)
Comment (solomontang) [reply src/containers/restitutionreferral/RestitutionReferral.js:206]: flow was really upset that it wasn't so i just threw them in there.

**Review by UnsungHero97 — APPROVED** [review]

**Review by giuliacampana — APPROVED** [review]

**Pull request merged**